pyproject.toml

```diff
@@ -1,6 +1,40 @@
+[tool.poetry]
+name = "pyquil"
+version = "2.28.0"
```

Comment:
I left this at 2.x.x, because I'm assuming Semantic Release will bump the major version when we go to release, based on having "Breaking" in our commit history.